### DISCUSSION TITLE
Will there be a schematic?

### DISCUSSION THREAD

**Discussion by TrokhymButko** [sh123/esp32_loradv#12] [Q&A] [answered]
I didn’t find a schematic diagram of where to solder the peripherals, mainly a question about the display. Or does it mean that this information needs to be obtained from the config file?

**Comment by sh123** [comment]
Hi, it is based on https://github.com/sh123/esp32_loraprs, which has schematics, additional peripherals are connected to two sets of pinouts on both sides of the board (a lot of soldering), correct pinouts should be checked from pinout config definition (from this project include/config.h). It is more a prototype, hence is does not have dedicated pinouts for each peripheral. I have plans to create dedicated board, but no free time for that yet.

**Reply by TrokhymButko** [reply]
There is no information in config.h where to connect the display, tell me where to connect the display.

**Reply by sh123** [reply, marked as answer]
Display is using I2C bus, so it should be connected to ESP32 SCL (IO22) SDA (IO21) pinouts, they are not declared, that's by default for Adafruit_SSD1306 library

![image](https://github.com/sh123/esp32_loradv/assets/1933302/507df200-aee4-4ac7-98bb-8cebce79e6c3)

**Reply by TrokhymButko** [reply]
Thank you, the display is already working. If it all works out
  on the training board, then I will wire the PCB myself.
![69](https://github.com/sh123/esp32_loradv/assets/108066880/ddf73edc-1e18-433a-9ff8-3c731a46d77b)
